.github/workflows/pr-validation.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ['7.4', '8.0', '8.1', '8.2']
+        php-versions: ['8.0', '8.1', '8.2', '8.3']
```

Comment:
Found a workaround in https://github.com/microsoft/kiota-abstractions-php/pull/122 so that we retain PHP 7.4 unit test runs